sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -119,6 +119,7 @@ export class Application extends RecordDataModel {
     {
       eager: false,
       cascade: true,
+      onDelete: "CASCADE",
```

Comment:
Why do we need this OnDelete when we have cascade:true. ?

Reply:
To make Typeorm delete the associated records that are not present in the array anymore. The items are being loaded and then they are replaced by the ones that must be kept in the association. The ones that are no longer in the array will be removed.